interop-tests/src/test/java/example/myapp/helloworld/grpc/RichErrorModelTest.java

```diff
@@ -52,8 +53,7 @@ private CompletionStage<ServerBinding> run(ActorSystem sys) throws Exception {
   public void testManualApproach() throws Exception {
     Config conf = ConfigFactory.load();
     ActorSystem sys = ActorSystem.create("HelloWorld", conf);
-    run(sys);
-
+    ServerBinding serverBinding = run(sys).toCompletableFuture().get(60, TimeUnit.SECONDS);
```

Comment:
I think 3 seconds should enough

Reply:
agree, but it's a timeout, so it's no problem if it's a little generous